registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -473,6 +475,7 @@
     "sha256:486162f59f1ce8fb54f15ceac5489441da2e5bab43804cfd6ac6707de55bfe67": ["v1.11.0-rc.1"]
     "sha256:c017edefd6eea25e0726a5fe3c35df3ca644814af69ddc9664119f8ca47d8dcc": ["v1.11.1"]
     "sha256:d8f52e5af431ef31d10ddda3ccb950ec4d0a57eb0d94831d6fe5cf9054f25c27": ["v1.11.2"]
+    "sha256:23d8cd2f75e2dff15fc659e5b539dbedc0e6e559d1459e740f2f7f34ff02f89a": ["v1.12.0-beta.0"]
```

Comment:
SHA `23d8cd2f75e2` matches kubeadm-bootstrap-controller v1.12.0-beta.0 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:23d8cd2f75e2dff15fc659e5b539dbedc0e6e559d1459e740f2f7f34ff02f89a/details).